pylint/checkers/variables.py

```diff
@@ -606,9 +606,25 @@ def leave_setcomp(self, _):
         # do not check for not used locals here
         self._to_consume.pop()
 
+    def _consume_in_scope(self, name, target_scope):
+        """move a name from to_consume to consumed in given scope
+        """
+        if not name:
+            return
+        for to_consume, consumed, scope in self._to_consume:
```

Comment:
I guess module scope can be only top-most enclosing scope, so shouldn't it be `self._to_consume[0]`?